library/ui-styles/src/main/res/values/colors.xml

```diff
@@ -141,6 +141,7 @@
 
     <!-- Shield colors -->
     <color name="shield_color_trust">#0DBD8B</color>
+    <color name="shield_color_trust_background">#0F0DBD8B</color>
```

Comment:
Have you tried to set an alpha on the ImageView instead of setting a dedicated tint color? It seems to be always the same applied alpha.

Reply:
We are using the red one in some fragment layout as background. So it is not possible to give a background alpha there.

Reply:
No problem, I just wanted to check.